packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -26,15 +21,7 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
   }
 
-  // If the destination is a UUID, assume it is an internal conversation ID.
-  if (UUID_REG.test(dest)) {
-    return dest;
-  }
-
-  // Next, assume it is a valid Hydra room ID.
-  // If decoding fails, it is a SIP or other destination.
-  // if the id is not present we will use the default destination
-  return deconstructHydraId(dest).id || dest;
```

Comment:
Are we removing this because we are going to use the convo URL, And will the convo ID in the convo Url need deconstruction as it could be public facing

Reply:
arun this was removed because it was broken and causing bugs. and we also cannot infer by convoId alone. so we are going to just use convo url